static/app/views/automations/components/dataConditionNodeList.tsx

```diff
@@ -26,6 +27,7 @@ interface DataConditionNodeListProps {
   onDeleteRow: (id: string) => void;
   placeholder: string;
   updateCondition: (id: string, condition: Record<string, any>) => void;
+  conflictingConditions?: string[];
```

Comment:
looks like its always provided in the two places this component is used. Prefer forcing the prop to be declared so it isn't accidentally omitted
```suggestion
  conflictingConditions: string[] | undefined;
```